internal/service/advancedclustertpf/model_ClusterDescription20240805.go

```diff
@@ -130,15 +138,15 @@ func NewReplicationSpecsObjType(ctx context.Context, input *[]admin.ReplicationS
 	return listType
 }
 
-func convertReplicationSpecs(ctx context.Context, input *[]admin.ReplicationSpec20240805, diags *diag.Diagnostics) *[]TFReplicationSpecsModel {
+func convertReplicationSpecs(ctx context.Context, input *[]admin.ReplicationSpec20240805, diags *diag.Diagnostics, apiInfo ExtraAPIInfo) *[]TFReplicationSpecsModel {
 	tfModels := make([]TFReplicationSpecsModel, len(*input))
 	for i, item := range *input {
 		regionConfigs := NewRegionConfigsObjType(ctx, item.RegionConfigs, diags)
 		tfModels[i] = TFReplicationSpecsModel{
 			Id:            types.StringPointerValue(item.Id),
 			ExternalId:    types.StringPointerValue(item.Id),
-			NumShards:     types.Int64Value(1),                         // TODO: Static
-			ContainerId:   conversion.ToTFMapOfString(ctx, diags, nil), // TODO: Static
+			NumShards:     types.Int64Value(1), // TODO: Static
```

Comment:
leftover comment

Reply:
TODOs will be handled as part of [the next Jira Issue](https://jira.mongodb.org/browse/CLOUDP-286481)